tutorials/Atmos/burgers_single_stack_fvm.jl

```diff
@@ -0,0 +1,709 @@
+# # Single stack tutorial based on the 3D Burgers + tracer equations
+
```

Comment:
This tutorial needs to be added to the tutorial list.

Reply:
Added